todo/src/App.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+import "./App.css";
+import { connect } from "react-redux";
+import { addTodo } from "./actions";
+import TodoList from "./components/TodoList";
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      newTodo: ""
+    };
+    this.addTodo = this.addTodo.bind(this);
+    this.updateNewTodo = this.updateNewTodo.bind(this);
+  }
+
+  addTodo(event) {
```

Comment:
```suggestion
  addTodo = (event) =>  {
```
If you write these methods as arrow functions you don't have to bind them in the constructor